feeds/src/config.ts

```diff
@@ -24,3 +24,63 @@ export interface OracleNode {
   name: string
   networkId: number
 }
+
+class UrlConfig {
+  static feedsJson(location: Location): string | undefined {
+    const query = new URLSearchParams(location.search)
+    const feedsJson = query.get('feeds-json')
+
+    if (feedsJson) {
+      return decodeURIComponent(feedsJson)
+    }
+    return undefined
+  }
+
+  static nodesJson(location: Location): string | undefined {
+    const query = new URLSearchParams(location.search)
+    const nodesJson = query.get('nodes-json')
+
+    if (nodesJson) {
+      return decodeURIComponent(nodesJson)
+    }
+    return undefined
+  }
+}
+
+export class Config {
+  static infuraKey(env = process.env): string {
```

Comment:
Do you think using a static getter would be more ergonomic?

Reply:
Getters can't take arguments

Reply:
We dont need arguments anymore since we're modifying `process.env` directly for tests now right?

Reply:
I kept the DI to keep them consistent as feeds + json still take `location`

Reply:
hm okay, i dont know how useful it is if its dead code though

Reply:
It's not really dead code though. The environment is dependency injected as an argument. I'm just keeping the pattern consistent across all functions.